src/style.css

```diff
@@ -1 +1,40 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  margin: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin: 0 4px 0 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('./images/star-active.svg');
+}
```

Comment:
In your code version, for each .stars--X class, you set the same styles for the corresponding .stars__star:nth-child(-n + X) element. This leads to redundant code repetition.
I recommend using the ``>`` direct descendant combinator.